### ISSUE TITLE
[Bug]: Screen Readers doesnt pick up `sp-progressbar`

### ISSUE THREAD

**Issue by Rajdeepc** [adobe/spectrum-web-components#4515]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Progress-bar

### Expected behavior

On tab stop on `sp-progressbar`, screen readers should be able to pickup the element content

### Actual behavior

No tab stop on `sp-progressbar`

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/progress-bar--label
2. Click on `Tab`
3. Open Voice Over Utility
4. try to focus on the Progressbar
5. See it cannot be focussed

### Sample code that illustrates the problem

```javascript
protected override firstUpdated(changes: PropertyValues): void {
    super.firstUpdated(changes);
    if (!this.hasAttribute('role')) {
        this.setAttribute('role', 'progressbar');
    }
   // this needs to be added
    if (!this.hasAttribute('tabindex')) {
        this.setAttribute('tabindex', '0');
    }
}

```

**Comment by blunteshwar** [comment]
Can you also add a relevant test for this.

**Comment by nikkimk** [comment]
@Rajdeepc this should be done with `aria-live` not a `tabindex` as progressbars are not interactive

**Comment by Rajdeepc** [comment]
Closing this since `aria-live` should take care of this!
https://www.magentaa11y.com/checklist-web/progress/
